snap/snapcraft.yaml

```diff
@@ -189,15 +189,20 @@ parts:
       # These are only included here until the OS snap stabilizes
       - libxml2
       - libpng12-0
+      - libmagickwand-6.q16-2
+      - libmagickcore-6.q16-2
     build-packages:
       - libxml2-dev
       - libcurl4-openssl-dev
       - libpng12-dev
-      - libjpeg9-dev
+      # This causes a conflict with libmagick* deps, that need libjpeg8-dev
+      #- libjpeg9-dev
```

Comment:
You can safely remove this comment, although I appreciate seeing it once to describe why the change was necessary, it doesn't need to stay in the YAML.